### DISCUSSION TITLE
How to handle issues/bugs with components/scopes?

### DISCUSSION THREAD

**Discussion by evelant** [teambit/bit#6890] [Help]
I had a problem with a component (https://github.com/teambit/bit/issues/6889) and looking at [the component on bit.cloud](https://bit.cloud/learnbit/extending-bit/metadata/component-todos/component-todos) I noticed that there's seemingly no standard way of handling/reporting issues with components.

Is there any sort of issue tracking/handling within bit? Is there a standard way to file issues against particular components?

It seems difficult at the moment since there isn't even a github link for a component unless you choose to add one to the readme.

**Comment by WillsterJohnson** [comment]
Did you find a solution for this? I can't seem to figure out where I'm supposed to report bugs for a component. Branches, packages, orgs, etc, all these GitHub things have an obvious working analog, but issues/discussions seem to be completely unaddressed? I can write in the feed of a scope, but as the URL suggests that is `.../~news`, not `.../~bug-reports`, `.../~feature-requests`, or `.../~discussions`

**Reply by ranm8** [reply]
Willster is right, Bit Cloud does not have an explicit issue reporting mechanism, you can use the feed to share comments and issues or use other systems for the matter.
